tests/utils/verification/volumeproperties.go

```diff
@@ -55,42 +55,37 @@ func GetVMAttachedToVolUsingAdminCli(volName string, hostname string) string {
 	return volProps[1]
 }
 
-// GetVolumePropertiesAdminCli returns capacity, attached-to-vm and disk-format field
-// for volume using Admin cli
-func GetVolumePropertiesAdminCli(volName string, hostname string) string {
-	cmd := admincli.ListVolumes + "-c volume,attached-to,capacity,disk-format 2>/dev/null | grep " + volName
-	op, _ := ssh.InvokeCommand(hostname, cmd)
-	if op == "" {
-		log.Fatal("Null value is returned by admin cli when looking for, size, disk-format and attached to vm. Output: ", op)
-	}
-	if len(strings.Fields(op)) != 4 {
-		log.Fatalf("Output is expected to consist of four elements only - "+
-			"volume name, attached-to-vm status, size and disk-format. Actual output: %s", op)
+// GetPropertiesForAllVolumesAdminCli returns a map of all the volumes from ESX and their corresponding
+// properties returned - capacity, attached-to-vm and disk-format
+func GetPropertiesForAllVolumesAdminCli(hostName string) map[string][]string {
```

Comment:
I strongly suggest to add a common util function to return a map of all volume properties in which the key is the property name and the value is the property value. That will be very useful to avoid such complicated function which is only useful for a specific test.

Can we file a issue to track this improvement?